app/renderer/components/navigation/urlBarSuggestions.js

```diff
@@ -36,12 +38,12 @@ class UrlBarSuggestions extends ImmutableComponent {
 
   render () {
     const suggestions = this.props.suggestionList
-    const bookmarkSuggestions = suggestions.filter((s) => s.type === suggestionTypes.BOOKMARK)
-    const historySuggestions = suggestions.filter((s) => s.type === suggestionTypes.HISTORY)
-    const aboutPagesSuggestions = suggestions.filter((s) => s.type === suggestionTypes.ABOUT_PAGES)
-    const tabSuggestions = suggestions.filter((s) => s.type === suggestionTypes.TAB)
-    const searchSuggestions = suggestions.filter((s) => s.type === suggestionTypes.SEARCH)
-    const topSiteSuggestions = suggestions.filter((s) => s.type === suggestionTypes.TOP_SITE)
+    const bookmarkSuggestions = suggestions.filter((s) => s.get('type') === suggestionTypes.BOOKMARK)
```

Comment:
Would it be better to just iterate `suggestionList` once (ex: w/ a forEach) and use a switch to push results onto empty array?